app/models.py

```diff
@@ -2145,3 +2145,23 @@ class VAProfileLocalCache(db.Model):
     __table_args__ = (
         UniqueConstraint('va_profile_id', 'communication_item_id', 'communication_channel_id', name='uix_veteran_id'),
     )
+
+
+class UserServiceRoles(db.Model):
+    __tablename__ = "user_service_roles"
+
+    id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4, nullable=False)
```

Comment:
I have no idea why the previous team made all the primary keys UUIDs.  It complicates making requests during testing.  Let's just use auto-incrementing integers for new models.  Please see VAProfileLocalCache for an example.

Reply:
So you would rather see it as 

```
id = db.Column(db.Integer, primary_key=True)
```

Instead of the UUID?